src/cascadia/LocalTests_TerminalApp/TestHostApp/Package.appxmanifest

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
+  <Identity Name="fba054a7-f1a1-4cb7-bb21-4949919af2f5" Publisher="CN=migrie" Version="1.0.0.0" />

```

Comment:
We can probably come up with our own catchy identity like `WindowsTerminal.TestHost` and set the publisher to, like, `CN=Windows Terminal Team` so when it spontaneously doesn't get unregistered on our boxes (this happens) it'll be fine

unless the identity needs to match something TAEF is expecting (!)

Reply:
Oh, I understand now what you did. Okay. I wasn't clear in my original.. okay. I get it. We can ship whatever, honestly, :smile: